Applications/Games/Subnautica/Steam/script.js

```diff
@@ -0,0 +1,31 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2013"]);
+include(["Engines", "Wine", "Verbs", "vcrun2008"]);
+include(["Engines", "Wine", "Verbs", "d3dx9_42"]);
+include(["Engines", "Wine", "Verbs", "xact"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+var screenSize = java.awt.Toolkit.getDefaultToolkit().getScreenSize();
+wine.setVirtualDesktop(screenSize.width, screenSize.height);
+
+new SteamScript()
+	.name("Subnautica")
+	.editor("Unknown Worlds Entertainment")
+	.author("Zemogiter")
+	.applicationHomepage("https://unknownworlds.com/subnautica/")
+	.wineDistribution("upstream")
+	.wineVersion(3.1)
+	.wineArchitecture("amd64")
+	.category("Games")
+	.preInstall(function(wine, wizard) {
+        	wine.vcrun2013();
+        	wine.corefonts();
+		wine.vcrun2008();
+		wine.d3dx9_42();
+		wine.xact();
+	})
+	.gameOverlay(false)
+	.appId(264710)
+	.executable("Steam.exe", ["-silent", "-applaunch", 264710, "-no-ces-sandbox"]) 
+	.go();
+	.print("Debug output");
```

Comment:
You probably want to remove the "Debug output" print out.

Reply:
Why remove it if we need info about what the script is doing?

Reply:
We do not want debug output in a finished script. If you need it during development, it's ok but it should not be committed.
If you believe that we need more printouts in general, we can think about it. However, in this case it should also not be part of the script but of the executing code.